.gitignore

```diff
@@ -1,3 +1,5 @@
 coverage
 .nyc_output
 /node_modules
+docs/index.html
```

Comment:
`docs` folder is used for the project documentation. Is there something like `dist` here?